src/components/App.js

```diff
@@ -24,7 +27,10 @@ const App = () => {
                 </ul>
             </nav>
             <Switch>
-                <Route path='/task01' component={Task01}/>
+                <Route path='/task01'>
+                    <Contact />
+                    <Home/>
```

Comment:
Tutaj miało to inaczej działać. Jeśli będzie w url `/task01/home` to tylko `Home` ma być widoczne ;)